app/src/main/res/drawable/splash_image.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <item android:drawable="@color/oppiaPrimaryLight" />
+
+  <item android:drawable="@drawable/splash_page" android:gravity="center"/>
+
+</layer-list>
```

Comment:
@anshbajpai
add a new line at EOF

Reply:
ok , @chrk2205 should I also remove the commented section inside splash_page.xml

Reply:
@chrk2205 I have done the required changes